samples/tock-sample-cas-auth-provider/README.md

```diff
@@ -0,0 +1,85 @@
+# tock-sample-cas-auth-provider
+
+Use this module as an example to create CAS authentication dedicated to your organization.
+
+## How it works
+
+Customer may want to provide a CAS based authentication to their Tock ecosystem.
+
+The intended way to do this is to ship a separated JAR containing a custom CAS authentication module.
+
+This way, customer specific security details could be committed in a separated repository
+
+## How to build
+
+In your `ẁorkspace/` directory:
+
+```shell
+# Creates directory
+mkdir my-company-cas
+
+# Copy tock sample code
+git clone https://github.com/theopenconversationkit/tock.git
+cp -r tock/samples/tock-sample-cas-auth-provider/. my-company-cas/
+
+# Now using version control
+cd my-company-cas
+git init
+```
+
+Using JDK 8+
+```shell
+mvn clean install
+```
+
```

Comment:
Add information to be careful to the path in `<relativePath>../../tock</relativePath>` in pom.xml according to where the user installed tock.